phpseclib/Crypt/Common/SymmetricKey.php

```diff
@@ -2629,4 +2638,8 @@ protected function createInlineCryptFunction($cipher_code)
 
         return \Closure::bind($func, $this, static::class);
     }
+    
+    public function setCustomPaddingString($str){
```

Comment:
This method should have [DocBlock comment](https://pear.php.net/manual/en/standards.sample.php)